app_dart/lib/src/service/branch_service.dart

```diff
@@ -134,12 +134,17 @@ class BranchService {
     required gh.RepositorySlug slug,
     required String branchName,
   }) async {
-    return (await githubService.getFileContent(
-      slug,
-      'bin/internal/release-candidate-branch.version',
-      ref: branchName,
-    ))
-        .trim();
+    String branchNameFromFile;
```

Comment:
Prefer using finals. We could rewrite this like

```dart
return githubService.getFileContent(
  slug,
  'bin/internal/release-candidate-branch.version',
   ref: branchName,
)
.then((String value) => value.trim())
.onError((_) => '');
```

https://dart.dev/tools/linter-rules/prefer_final_locals#:~:text=DO%20prefer%20declaring%20variables%20as,the%20compiler%20to%20do%20optimizations.

Reply:
Synced with Casey and will return empty string, because other people might be able to create a branch with the nonempty name, and gain release branch privilege.